python/pyspark/sql/functions.py

```diff
@@ -1839,15 +1839,19 @@ def from_json(col, schema, options={}):
     string.
 
     :param col: string column in json format
-    :param schema: a StructType or ArrayType of StructType to use when parsing the json column
+    :param schema: a StructType or ArrayType of StructType to use when parsing the json column.
     :param options: options to control parsing. accepts the same options as the json datasource
 
+    .. note:: Since Spark 2.3, the DDL-formatted string is also supported for ``schema``.
```

Comment:
Now, it looks as below:

<img width="580" alt="2017-07-02 4 57 55" src="https://user-images.githubusercontent.com/6477701/27765087-ef86dbaa-5ee3-11e7-86ac-bbc5d61adc3c.png">